lib/sdn_epc/forwarder.ex

```diff
@@ -8,43 +8,48 @@ defmodule SdnEpc.Forwarder do
   and controller.
   """
 
+  defstruct([:datapath_id])
+
   # Client API
 
   @doc """
   Starts SdnEpc.Forwarder process.
   """
-  @spec start_link() :: GenServer.on_start()
-  def start_link() do
-    GenServer.start_link(__MODULE__, :ok, name: __MODULE__)
+  @spec start_link(name :: atom()) :: GenServer.on_start()
+  def start_link(name) do
+    GenServer.start_link(__MODULE__, :ok, name: name)
```

Comment:
The name doesn't have to be atom. You can use the [Registry](https://hexdocs.pm/elixir/Registry.html).